lib/testScenario/reportGenerator.ts

```diff
@@ -252,6 +401,19 @@ export class ReportGenerator {
     return [];
   }
 
+  private getRuntimeError(it: RawExecution): RuntimeError {
+    const ret: RuntimeError = {
+      code: "",
+      message: "",
+      severity: "Error",
+      detail: this.dataMasker.jsonStringify(it.response.body),
+    };
+    const responseObj = this.dataMasker.jsonParse(it.response.body);
+    ret.code = responseObj?.error?.code;
```

Comment:
Is it possible to set with these values?
code = `RUNTIME_ERROR_${statusCode}`,
message = "code:xxx, message:xxx".